presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java

```diff
@@ -92,4 +92,18 @@ public NodeState getServerState()
             return ACTIVE;
         }
     }
+
+    @GET
+    @Path("coordinator")
+    @Produces(TEXT_PLAIN)
+    public Response getServerCoordinator()
+    {
+        if (this.serverInfo.isCoordinator()) {
```

Comment:
I tired `204`, but the load balancer would only send traffic for `200`.

I have updated the response to be `200` (OK) and `404` (Not Found) if not a coordinator.